code/css/index.css

```diff
@@ -49,10 +52,14 @@ footer {
 }
 
 .menu li:hover {
-	color: white;
+	background-color: #111;
+}
+
+.active {
```

Comment:
I think this line should be changed to: 
`.menu li:active`
Since there is no use of the class name `active` anywhere in the HTML or JS code, this background color will never be applied.